specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/applicationGateway.json

```diff
@@ -1319,6 +1319,18 @@
           "type": "integer",
           "format": "int32",
           "description": "Capacity (instance count) of an application gateway."
+        },
+        "family": {
```

Comment:
>family [](http://example.com/codeflow?start=9&length=6)

Adding a new property without changing the API version is a breaking change.

Reply:
There is an API version in previous NRP release. Here is the code change in NRP:https://msazure.visualstudio.com/One/_git/Networking-nrp/commit/e7cba6645bb14e3611b339a4f12dad37110e7f14?_a=contents&path=/src/sources/Frontend/FrontendContract/Csm.Public/ApplicationGatewaySkuFamily.cs